pytest_timeout.py

```diff
@@ -38,6 +38,7 @@
 used in the test.
 """.strip()
 
+KNOWN_DEBUGGING_MODULES = {"pydevd", "ipdb"}
```

Comment:
Right, this is great that this method can also work for other debuggers.  That's what we wanted!  I think we should certainly also support plain stdlib pdb :)  I just checked and believe the stdlib `pdb.set_trace()` installs `bdb.trace_dispatch`, i.e. we should add `bdb` here too.

Reply:
I just checked ipdb as well and I also get `bdb` for it.  Here's how I checked (this could be the wrong way):
```
[sc]flub@powell /t/sandbox
> cat test.py                                                                                                                                        3ms
import ipdb

ipdb.set_trace()
[sc]flub@powell /t/sandbox
> python3 test.py                                                                                                                                    1ms
--Return--
None
> /tmp/sandbox/test.py(3)<module>()
      1 import ipdb
      2
----> 3 ipdb.set_trace()

ipdb> import sys
ipdb> sys.gettrace().__module__
'bdb'
```